mbedtls/src/pk/mod.rs

```diff
@@ -38,6 +38,8 @@ pub use crate::ecp::EcGroup;
 // SHA-256("Fortanix")[:4]
 const CUSTOM_PK_TYPE: pk_type_t = 0x8b205408 as pk_type_t;
 
+const RAW_RSA_DECRYPT : i32 = 2147483647;
```

Comment:
Please use a random value
```suggestion
const RAW_RSA_DECRYPT : i32 = 1040451858;
```